code/modules/mob/living/simple_animal/vox.dm

```diff
@@ -22,15 +22,11 @@
 	var/amp = null
 	var/quills = 3
 
-/mob/living/simple_animal/vox/armalis/Die()
-
-	living_mob_list -= src
-	dead_mob_list += src
-	stat = DEAD
-	visible_message("<span class='danger'><B>[src] shudders violently and explodes!</B>","<span class='warning'>You feel your body rupture!</span></span>")
-	explosion(get_turf(loc), -1, -1, 3, 5)
-	src.gib()
-	return
+/mob/living/simple_animal/vox/armalis/death(var/gibbed=FALSE)
+	if (!gibbed)
+		visible_message("<span class='danger'><B>[src] shudders violently and explodes!</B>","<span class='warning'>You feel your body rupture!</span></span>")
```

Comment:
Well for one thing, they get gibbed so `death()` would be called again with `gibbed=TRUE` so it'd NEVER run.

Though I realize I forgot a super call.